mesonbuild/compilers/mixins/clang.py

```diff
@@ -56,6 +57,14 @@ def __init__(self):
     def get_colorout_args(self, colortype: str) -> T.List[str]:
         return clang_color_args[colortype][:]
 
+    def has_builtin_define(self, define: str) -> bool:
+        return define in self.defines
+
+    def get_builtin_define(self, define: str) -> T.Optional[str]:
+        if define in self.defines:
+            return self.defines[define]
+        return None
```

Comment:
```suggestion
        return self.defines.get(define)
```

Same thing, just less code.